cp-remote

```diff
@@ -416,6 +430,11 @@ case $COMMAND in
         validate_settings
         build
         ;;
+    bash)
```

Comment:
Maybe but they do call different functions so that you get the deprecation warning when using `ssh`